### PR TITLE
Questions step: Only prefill customer name for first ticket

### PR DESCRIPTION
**Pull request by raphaelm**
see internal Ticket# 2396506

### CODE REVIEW

**Comment by wiffbi** [comment]
There is one edge-case where this does not work: when I have a non-admission product as the first product in my basket, it does not auto-fill the first admission-product with the name. Probably very very rare and so rare enough to be ignored?

File: src/pretix/base/views/mixins.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def forms(self):
                 ))
             )
 
-            override_sets = self.get_question_override_sets(cr)
+            override_sets = self.get_question_override_sets(cr, idx)
```
Comment (wiffbi) [inline src/pretix/base/views/mixins.py:99]: Based on idx, there is one edge-case where this does not work: when I have a non-admission product as the first product in my basket, it does not auto-fill the first admission-product with the name. Probably very very rare and so rare enough to be ignored?

**Review by wiffbi — COMMENTED** [review]
Looks good, just a comment regarding an edge-case having a non-admission-product as the first product in the cart.

File: src/pretix/base/views/mixins.py (hunk 1 above)
Comment (raphaelm) [reply src/pretix/base/views/mixins.py:99]: Yeah, I think that's fine.

**Pull request merged**